WooCommerce/Classes/ViewRelated/Products/Edit Product/Inventory Settings/ProductInventorySettingsViewController.swift

```diff
@@ -0,0 +1,295 @@
+import UIKit
+import Yosemite
+
+final class ProductInventorySettingsViewController: UIViewController {
+
+    @IBOutlet private weak var tableView: UITableView!
+
+    // Editable data - shared.
+    //
+    private var sku: String?
+    private var manageStockEnabled: Bool
+    private var soldIndividually: Bool
+
+    // Editable data - manage stock enabled.
+    private var stockQuantity: Int?
+    private var backordersSetting: ProductBackordersSetting?
+
+    // Editable data - manage stock disabled.
+    private var stockStatus: ProductStockStatus?
```

Comment:
nit: comment formatting `///`

Reply:
Generally, I use `///` for property/function/struct/class level comments and `//` for comments on a block of properties or misc notes. Lemme know if there are some official/unofficial docs for best practices!